### PR TITLE
Use clang-format-13

### PR DESCRIPTION
**Pull request by rleh**
In #21 I added `clang-format`, but wasn't aware the default in Ubuntu 20.04 is only version 10.

### CODE REVIEW

**Review by chris-durand — APPROVED** [review]

**Comment by rleh** [comment]
@chris-durand @salkinium Are you both ok with using the clang-format-13 from the external LLVM repo?

**Comment by chris-durand** [comment]
> @chris-durand @salkinium Are you both ok with using the clang-format-13 from the external LLVM repo?

Of course, I suggested using the the latest LLVM stable version in ~~#22~~ https://github.com/modm-ext/docker-modm-build/pull/21#issuecomment-1008357648.

**Comment by chris-durand** [comment]
@rleh I am just curious, why did you decide on using version 13 instead of the most recent default Ubuntu package you were favouring previously?

File: avr.Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/modm-ext/modm-build-base:latest
+FROM ghcr.io/modm-ext/modm-build-base:TAG
 LABEL maintainer="Niklas Hauser <niklas.hauser@rwth-aachen.de>, Raphael Lehmann <raphael+docker@rleh.de"
```
Comment (chris-durand) [inline avr.Dockerfile]: There is a missing ">" in the maintainer label, you could fix that as well if you like. Same for the RISC-V docker file.

**Review by chris-durand — APPROVED** [review]

**Comment by rleh** [comment]
> @rleh I am just curious, why did you decide on using version 13 instead of the most recent default Ubuntu package you were favouring previously?

"latest is greatest" 😄

There are also some interesting [config options new in 13](https://clang.llvm.org/docs/ClangFormatStyleOptions.html).

File: avr.Dockerfile (hunk 1 above)
Comment (rleh) [reply avr.Dockerfile]: Done: 2490e4b

**Review by salkinium — APPROVED** [review]

**Pull request merged**